app/src/main/java/gov/hhs/cdc/trustedintermediary/external/hapi/HapiLabOrderConverter.java

```diff
@@ -44,21 +45,23 @@ private HapiLabOrderConverter() {}
     @Override
     public LabOrder<Bundle> convertToOrder(final PatientDemographics demographics) {
         logger.logInfo("Converting demographics to order");
-
+        var orderDateTime = Date.from(Instant.now());
         var labOrder = new Bundle();
         var labOrderId = UUID.randomUUID().toString();
         labOrder.setId(labOrderId);
         labOrder.setIdentifier(new Identifier().setValue(labOrderId));
         labOrder.setType(Bundle.BundleType.MESSAGE);
-        labOrder.setTimestamp(Date.from(Instant.now()));
+        labOrder.setTimestamp(orderDateTime);
 
         var patient = createPatientResource(demographics);
-        var serviceRequest = createServiceRequest(patient);
+        var serviceRequest = createServiceRequest(patient, orderDateTime);
         var messageHeader = createMessageHeader();
```

Comment:
Just checking, will you be adding the `meta` field to the `MessageHeader` resource in this PR or a future PR?  That was also a new addition to to the `lab_order.json`.